### ISSUE TITLE
Ctrl + Click for Definitions doesn't appear to work for 2nd and 3rd extended classes

### ISSUE THREAD

**Issue by addshore** [DEVSENSE/phptools-docs#65]
In a class that looks like this:

```
interface PrefetchingTermLookup extends TermBuffer, TermLookup, AliasTermBuffer {
}
```

Ctrl + Clicking on the first extension class correctly works:

![image](https://user-images.githubusercontent.com/3308769/89530841-18b2a700-d7e7-11ea-98a6-1926d8b03bdd.png)

However ctrl + clicking on the other 2 classes does not do anything :/

**Comment by jakubmisek** [comment]
fixed since `1.0.5019`+ (to be released)

**Comment by addshore** [comment]
I can verify this is working for me :)
